### DISCUSSION TITLE
Keeping my JPEGView.ini clean and in sync

### DISCUSSION THREAD

**Discussion by slovuj** [sylikc/jpegview#115] [General]
Regarding the JPEGView.ini file in the kit:

1. I have the following line in mine:
`FilesProcessedByWIC=*.wdp;*.hdp;*.jxr;*.ico;*.pic`

I added **`*.ico;*.pic`** over time; does it make sense to others to update the above
line in the kit version? I'd like to avoid constant merging...

2. In my version, I have this line as:
`ColorCorrection = "R: 0.2 G: 0.1 B: 0.35 C: 0.1 M: 0.3 Y: 0.15" `
while yours is without quotes. Which is "better"?

Thanks for the effort of keeping this jewel alive!

Slo

**Comment by sylikc** [comment]
> 1. I have the following line in mine:
>    `FilesProcessedByWIC=*.wdp;*.hdp;*.jxr;*.ico;*.pic`
> 
> I added **`*.ico;*.pic`** over time; does it make sense to others to update the above line in the kit version? I'd like to avoid constant merging...

Actually right now even when i put *.ico in for testing, does it load the files into the filelist to open?  I'm still testing, there was another issue open about ICO support #42

> 2. In my version, I have this line as:
>    `ColorCorrection = "R: 0.2 G: 0.1 B: 0.35 C: 0.1 M: 0.3 Y: 0.15" `
>    while yours is without quotes. Which is "better"?

So, it should be without quotes.  It was merged as part of a PR #12 that quotes was causing the setting to not be honored

**Reply by slovuj** [reply]
1. I can view .ico files without problems when I do this:

`JPEGView.exe C:\Windows\SystemApps\Microsoft.Windows.SecHealthUI_cw5n1h2txyewy\Assets\*.ico`

But if I hit 'w', it opens Windows Explorer, not the internal file manager. Is this what you are asking?

2. Thanks, I removed the quotes.

**Comment by sylikc** [comment]
closing this discussion and referring it to a defaults issue #140
